riscv/src/riscv_asm.rs

```diff
@@ -0,0 +1,2563 @@
+// auto-generated: "lalrpop 0.19.9"
```

Comment:
is this file supposed to go in the repo?

Reply:
good catch